x-pack/plugins/reporting/export_types/printable_pdf/server/lib/generate_pdf.js

```diff
@@ -11,7 +11,7 @@ import { pdf } from './pdf';
 import { groupBy } from 'lodash';
 import { oncePerServer } from '../../../../server/lib/once_per_server';
 import { screenshotsObservableFactory } from './screenshots';
-import { getLayoutFactory } from './layouts';
+import { getlayout } from './layouts/layout_factory';
```

Comment:
`getlayout` => `getLayout`.  actually, I'd name is `createLayout` since it's creating a new object, not getting an existing one.  (Though I know the current code uses `getLayout` so if that ends up requiring a large refactor, fine to leave as is)